docs/monthly-meeting/2022-10.md

```diff
@@ -0,0 +1,91 @@
+# Documentation Community Team Meeting (October 2022)
+
+- **Date:** 2022-10-03
+- **Time:** [19:30 UTC](https://arewemeetingyet.com/UTC/2022-10-03/19:30/Docs%20Meeting)
+- **This HackMD:** https://hackmd.io/@encukou/pydocswg1
+- [**Discourse thread**](https://discuss.python.org/t/19070) for October
+- **Calendar for future meetings:** (send your e-mail to Mariatta for a calendar invitation)
+- **How to participate:**
+  -  Go to [Google Meet](https://meet.google.com/dii-qrzf-wkw) and ask to be let in.
+  -  To edit notes, click the “pencil” or “split view” button on the [hackMD document]
+  -  (https://hackmd.io/@encukou/pydocswg1). You need to log in (e.g. with a GitHub account).
+
+By participating in this meeting, you are agreeing to abide by and uphold the [PSF Code of Conduct](https://www.python.org/psf/codeofconduct/).
+Please take a second to read through it!
+
+## Roll call
+
+- Petr Viktorin / `@encukou`
+- Adam T / `@AA-Turner`
+- Ezio Melotti / `@ezio-melotti`
+- Hugo van Kemenade / `@hugovk`
+- Mariatta / `mariatta`
+- Joannah / `nanjekyejoannah`
+- Alex Waygood / `@AlexWaygood`
+- CAM Gerlach / `@CAM-Gerlach`
+- Pradyun / `@pradyunsg`
+- Leam Hall
+
+## Quick updates - Introductions
+
+> 60 second updates on things you have been up to, questions you have, or developments you think people should know about. Please add yourself, and if you do not have an update to share, you can pass.
+
+- Videos for the Diataxis workshop were published!  https://discuss.python.org/t/recordings-available-for-python-docs-diataxis-workshop/19518
+
+
+## Discussion
+
+* (CAM) Should we enable Intersphinx support on the CPython 3.12 docs? It would primarily benefit porting/syncing docs from other places, like PEPs, importlib metadata/resources, and packaging, but could cause problems for downstreams (e.g. Linux distros) building the docs from source without network access. See [python/cpython#97785](https://github.com/python/cpython/pull/97785/).
+
+   * Allows single source of truth for definitions, etc.
+   * Not offline buildable
+   * Petr: would like Sphinx to provide a directory to reference local inv files to do offline builds
+   * next steps: Cam to open a PR in 3.12, also a Discuss thread. Petr to open an issue in Sphinx.
+
+* Docs to support Sphinx 5.x. @CAM-Gerlach wants to add a CI workflow during the sprint week for the oldest-supported version of Sphinx.
+    * Adam will work on sphinx support on python-docs-theme
+
+* (Ezio) Should we enable the default role and alias it to `:literal:` (`` `...` ``) for docs.python.org?
+  * Less typing required, and no more errors caused by MarkDown-like `` `...` `` in rst documents
+  * It's not used for anything, so we could use it instead of (or as an alias to) ` ``...`` `
+  * Unless we mass edit the docs (to switch to single backticks), it might lead to inconsistencies
+  * We could also alias it to `:any:`?
```

Comment:
```suggestion
  * Unless we mass edit the docs (to switch to single backticks), it might lead to inconsistencies

If enabled, what should it be aliased to?
  * We could also alias it to `:any:`?
```